src/rgen/Microsoft.Macios.Generator/Emitters/BindingSyntaxFactory.Property.cs

```diff
@@ -41,10 +45,65 @@ internal static (string? Getter, string? Setter) GetObjCMessageSendMethods (in P
 		return default;
 	}
 
+	internal static (StatementSyntax Send, StatementSyntax SendSupper) GetGetterInvocations (in Property property,
+		string? selector, string? sendMethod, string? superSendMethod)
+	{
+		// if any of the methods is null, return a throw statement for both
+		if (selector is null || sendMethod is null || superSendMethod is null) {
+			return (ThrowNotImplementedException (), ThrowNotImplementedException ());
+		}
+
+		// the invocations depend on the property requiring a temp return variable or not
+		if (property.UseTempReturn) {
+			return (
+				Send: ThrowNotImplementedException (),
+				SendSupper: ThrowNotImplementedException ()
```

Comment:
Sending supper sounds good, but this might be more correct:

```suggestion
				SendSuper: ThrowNotImplementedException ()
```